src/client/extension.ts

```diff
@@ -15,86 +13,13 @@ export function activate(context: vscode.ExtensionContext) {
   let help = vscode.commands.registerCommand('v.help', commands.help);
   let ver = vscode.commands.registerCommand('v.ver', commands.ver);
   let path = vscode.commands.registerCommand('v.path', commands.path);
-  let testFile = vscode.commands.registerCommand(
-    'v.test.file',
-    commands.testFile
-  );
-  let testPackage = vscode.commands.registerCommand(
-    'v.test.package',
-    commands.testPackage
-  );
-  let playground = vscode.commands.registerCommand(
-    'v.playground',
-    commands.playground
-  );
-
+  let testFile = vscode.commands.registerCommand('v.test.file', commands.testFile);
```

Comment:
Also, you could get rid of result values, as they aren't used actually:
```ts
vscode.commands.registerCommand('v.test.file', commands.testFile);
```

Reply:
@alexesprit Yeah... I think so...